solr-ingest/src/test/java/edu/unc/lib/dl/data/ingest/solr/filter/SetDescriptiveMetadataFilterTest.java

```diff
@@ -267,7 +279,11 @@ public void testMultipleLanguages() throws Exception {
     @Test
     public void noMODS() throws Exception {
         when(idb.getTitle()).thenReturn(null);
-        when(dip.getLabel()).thenReturn("test label");
+        when(objResc.hasProperty(DcElements.title)).thenReturn(true);
+        Statement titleStmt = mock(Statement.class);
```

Comment:
I was only using it in one test so I didn't feel the need to have it initialized for all the other tests too